recipes/CImg/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set name = "CImg" %}
+{% set version = "2.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dtschump/CImg/archive/v.{{ version }}.tar.gz 
+  sha256: 3ad10cb8842e945af208a05b8ed36f4428317ef754c6e341e4ca6ac70b8a8540
+
+build:
+  noarch: ''
+  noarch_python: false
+  number: 0
+
+requirements:
+  build:
+  host:
+  run:
+
+test:
+  imports:
```

There should always be a test even if it's a simple existence test.
```suggestion
  commands:
    - test -f something
```

---

test added